base/sno/kustomization.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - namespace.yaml
+  - pullsecret-sealed.yaml
+  - aci.yaml
+  - clusterdeployment.yaml
+  - infraenv.yaml
+  - klusterletaddon.yaml
+  - managedcluster.yaml
+  - bmc-secrets.yaml
+  - bmh.yaml
+  - nmstate-config.yaml
+#  - provisioningoverride.yaml
```

Comment:
I see the file included in this PR, should it be in a comment?

Reply:
I uncommitted it in the base because that is how it should be. 
The parameter of concern is disableVirtualMediaTLS: true whether we want this enabled or disabled and that is depended on the hardware. I will create a patch to either have this enabled or disabled in the overlay...

Reply:
Could you please add a readme to the base explaining when to use `disableVirtualMediaTLS: true`?

Reply:
I added it